packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC, ReactNode } from 'react';
+import { cx } from 'emotion';
+import { useTheme } from '../../themes';
+import { getTabsStyle } from './styles';
+
+export interface TabProps {
+  label: string;
+  active?: boolean;
+  icon?: ReactNode;
```

Comment:
WDYT about using `IconType` instead of node? I doubt there will be ever a need for icons to be interactive, so maybe we could just hide Icon renderin in the Tab component